sktm/__init__.py

```diff
@@ -124,7 +123,6 @@ def add_pw(self, baseurl, pname, lpatch=None, restapi=False, apikey=None,
                 baseurl, pname, int(lpatch) if lpatch else None, skip
             )
 
-            # FIXME Figure out the last patch first, then create the interface
```

Comment:
I didn't read this code now, but I think converting between project ID and name shouldn't be a job for a patchwork project class, but rather for a patchwork server (instance) class. The latter can have an interface to retrieve project objects for specific IDs and have an interface for conversion to and from a project name, i.e. the `_get_project_id()` function should be moved there. Then we will be able to deal with these FIXMEs properly.

That's not something we need to do in this PR, though.

Reply:
Having a class to deal with a full Patchwork server is definitely a neat idea. Do you think you can submit an issue for that, so we don't forget about it while dealing with other things?

Reply:
Aand, here it is: #140.